Source/BoundaryConditions/PML.cpp

```diff
@@ -714,9 +717,10 @@ PML::PML (const int lev, const BoxArray& grid_ba, const DistributionMapping& gri
     }
 
     Box single_domain_box = is_single_box_domain ? domain0 : Box();
+    amrex::Real m_v_sigma_sb = v_sigma_sb;
```

Comment:
I think it could be better to declare this variable `m_v_sigma_sb` in PML.H as a member variable of the class `PML` (probably `private` would work, such as for other variables like `m_dive_cleaning`) and then initialize it in the [initializer list](https://en.cppreference.com/w/cpp/language/constructor) of the `PML` constructor in PML.cpp (where `m_dive_cleaning` and other similar variables are initialized). With all of this, the line here would not be needed anymore. Could you try to see if this also works?